config.go

```diff
@@ -194,6 +191,14 @@ func (conf *Config) prepareEnvironment() error {
 		}
 	}
 
+	// Check if servers in CarbonAddrs are resolvable
+	for _, carbonAddr := range conf.CarbonAddrs {
+		_, err := net.ResolveTCPAddr("tcp", carbonAddr)
+		if err != nil {
+			return errors.New("This is not a valid address in CarbonAddrs: " + err.Error())
```

Comment:
wrong error message. the addresses can still be valid even when dns is down. a proper message would be something like `could not resolve`

Reply:
Fixed, `Could not resolve an address from CarbonAddrs:` now